test_netcdf_timeaverage.jl

```diff
@@ -0,0 +1,213 @@
+using Revise
```

Comment:
`Revise` shouldn't be here anymore, correct?

Reply:
I agree!